decidim/decidim
# Add correct call for conference speaker

#### :tophat: What? Why?
As per #10027 we can see the conference speaker is not being defined. I have looked into the issue and i have seen that edit action is referring to conference speaker as `@item`, and the the selected_meetings helper was checking `@current_speaker` variable. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #10027

#### Testing
1. Sign in as admin
2. Go to admin dashboard, conference, speakers, create a new speaker
3. Fill in the required fields with any participant type
4. Select a related meeting
5. Save the form
6. Edit the same speaker
7. See that related meetings as being highlighted

### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![image](https://user-images.githubusercontent.com/105683/201281549-241f82a0-0dbc-41f1-b1fa-9ea61c7f6d43.png)


:hearts: Thank you!
